istioctl/cmd/describe.go

```diff
@@ -143,6 +147,13 @@ the configuration objects that affect that pod.`,
 				return err
 			}
 
+			// render PeerAuthentication info
+			fmt.Fprintf(writer, "--------------------\n")
+			err = describePeerAuthentication(writer, configClient, istioNamespace, ns, k8s_labels.Set(pod.ObjectMeta.Labels))
```

Comment:
is the `istioNamespace` the root namespace? Note it could be different and configurable in meshconfig.

Reply:
Updated to use root namespace.